src/Control/Monad/Eff/Console/Unsafe.purs

```diff
@@ -0,0 +1,13 @@
+-- | Dodgy functions to output runtime representation of arbitrary values.
```

Comment:
s/Dodgy/Unsafe/ makes sense.

Is something technically wrong with "runtime representation"? It seems like a good explanation of what it does.